docs/source/quantization.rst

```diff
@@ -1000,6 +1000,116 @@ Custom API Example::
   mq = torch.ao.quantization.quantize_fx.convert_fx(
       mp, convert_custom_config=convert_custom_config_dict)
 
+Deciding Which Quantization Tools to Use for Your Use Case
+----------------------------------------------------------
+
+As we've described above, the quantization codebase equips users with a number of tools and options to quantize their models. These include
+
+* The choice of quantization mode
+
+  * Eager Mode Quantization (Eager Mode)
+  * Fx Graph Mode Quantization (Fx)
+
+* When/How to apply quantization
+
+  * Post Training Quantization (PTQ)
+  * Quantization Aware Training (QAT)
+
+* Quantized Ops Type
+
+  * Dynamic Quantization
+  * Static Quantization
+  * Weight Only Quantization
+
+* QScheme
+
+  * per_tensor_affine
+  * per_tensor_symmmetric
+  * per_channel_affine
+  * per_channel_symmetric
+
+Beyond this, the various quantization ops, modes and flows can be mixed and
+matched to different layers in the model leading to a truly staggering number of options. Given the difficulty of navigating this
+decision space, this section will focus on how to decide which flows and techniques to use in your particular use case.
+
+Eager Mode Quantization vs Fx Graph Mode Quantization:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The first question is whether to use Eager Mode or Fx. Both modes largely attempt to do
+the same thing, but obtain certain structural information about the model in different ways. Eager Mode requires
+the user to manually specify where to add transitions between quantized and non-quantized dtypes, whereas Fx
+traces the model in order to learn the model's structure and automatically determine where to place these transitions.
+
+In general, which you should use mostly boils down to 2 things, feature availability and ease of use, both of which lean in
```

Comment:
"ease of use" does not favor FX graph mode quant, unfortunately :(

Reply:
i meant like how much manual stuff there is, i'll rephrase

Reply:
also do you disagree with the conclusion that users should see if FX works for them as a first step?